ocs_ci/deployment/gcp.py

```
@@ -68,11 +79,127 @@ def check_cluster_existence(self, cluster_name_prefix):
+                cco.extract_credentials_requests(
+                    release_image,
+                    install_config,
+                    pull_secret_path,
+                    credentials_requests_dir,
+                )
+            cco.delete_gcp_sts_resources(
+                self.cluster_name,
+                gcp_project,
+                credentials_requests_dir,
+            )
+        super(GCPIPI, self).destroy_cluster(log_level)
```

Ruff (0.15.15)

[warning] Use `super()` instead of `super(__class__, self)`

Remove `super()` parameters

(UP008)